doc/00-onboarding-guide.md

```diff
@@ -1,30 +1,31 @@
+> ** we are improving the process here with Azure CLI team soon 
 
+## Step 2. Quick Validation
+If it is your first time to use Azure Cli code-gen, please discuss with the contact person. He/She will: 
 1. Do a quick validation on your swagger and prepare initial readme files for you
-2. Have one person assigned for your service to follow up with your for any questions and support needed
-
-## Step 2. Generate code and give it a try
+2. follow up with you for any questions and support needed during the service onboarding
 
+## Step 3. Generate code and give it a try
 1. There are 3 ways to generate code now, you can pick the one you prefer. Details can be found at [here](how-to-generate.md)
 2. Try the generated command to see whether it meets your need. Further customization is supported if needed. Detail can be found at [FAQ](faq.md).
 
-## Step 3. Run Test
-1. Test will also be generated based on the examples in swagger. Run these test and make sure all of them can pass. (Refer to [here](how-to-generate.md#through-our-pre-prepared-docker) for how to run test). Customization is also supported for test if needed. Detail can be found at [here](04-scenario-test-configuration.md)
+## Step 4. Run Test
+1. Test will also be generated based on the examples in swagger. Run these tests and make sure all of them can pass. (Refer to [here](how-to-generate.md#through-our-pre-prepared-docker) for how to run test). Customization is also supported for test if needed. Detail can be found at [here](04-scenario-test-configuration.md)
 2. Add more test if the generated test is not enough. Detail can be found at [here](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md)
 
-## Step 4. Onboarding to azure-cli
-1. Follow [azure-cli guidance](https://github.com/Azure/azure-cli/blob/dev/doc/onboarding_guide.md) to onboard to azure-cli
-> ** we are improving the process here with Azure CLI team now
-
-## Step Next. CLI is ready when swagger is ready
-Autorest.az has been integrated into swagger PR. It means you can start reviewing your CLI code/command immediately when your swagger PR is filed for your new features/changes, so your new features/changes can have Azure CLI support very soon. 
-
-Hopefully, **your CLI is ready to merge/release when your swagger is ready to merge. :shipit: :+1:** 
+## Step 5. Run Linter and Style
+1. Run linter and style check for the code generated and make sure all of them can pass. (Refer to [[here](how-to-generate.md#through-our-pre-prepared-docker) for how to run linter and style check])
 
-Check this [10-minutes video](https://msit.microsoftstream.com/video/71cea3ff-0400-a9f4-01b4-f1ea9e9b130e) for more details
-> ** we are working on the process here with Azure CLI and swagger team now
+## Step 6. Code review with Cli team
+1. Discusses with your contact person to ask for code review of the Az Cli code.
 
+## Step 7. Onboarding to azure-cli
```

Comment:
>Onboarding to azure-cli [](start = 11, length = 23)

please merge this with Step6

Reply:
resolved

---
In reply to: [449504853](https://github.com/Azure/autorest.az/pull/461#discussion_r449504853) [](ancestors = 449504853)